### PR TITLE
Make the sendmsg tests more resillient to a large kernel buffer

### PR DESCRIPTION
**Pull request by alex**
## Contributor Checklist:

* [x] There is an associated ticket in Trac. Create a new one at https://twistedmatrix.com/trac/newticket
* [x] The changes pass minimal style checks (see: https://twistedmatrix.com/trac/wiki/TwistedDevelopment#GettingYourPatchAccepted )
* [x] I have created a newsfragment in src/twisted/newsfragments/ (see: https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles )
* [x] I have updated the automated tests.

### CODE REVIEW

**Comment by alex** [comment]
Confirming that this fixes it on the cryptography side here: https://travis-ci.org/pyca/cryptography/jobs/379785983

File: src/twisted/python/test/test_sendmsg.py [hunk 1]
```diff
@@ -397,7 +397,7 @@ def test_shortsend(self):
         """
         L{send1msg} returns the number of bytes which it was able to send.
         """
-        message = "x" * 1024 * 1024
+        message = "x" * 1024 * 1024 * 8
```
Comment (moshez) [inline src/twisted/python/test/test_sendmsg.py:400]: Please update the comment below to say "8MB" (or to not mention the size and to refer to the code).

**Review by moshez — APPROVED** [review]
Please update comments and merge

**Comment by hawkowl** [comment]
This needs to be ran on the full suite, with a twisted-compatible branch name, @moshez  :)

**Comment by alex** [comment]
@hawkowl travis is green -- can you do that stuff?

**Pull request merged**